eth/api.go

```diff
@@ -694,6 +695,12 @@ func (s *PublicBlockChainAPI) doCall(args CallArgs, blockNr rpc.BlockNumber) (st
 	}
 	stateDb = stateDb.Copy()
 
+	// If there's no code to interact with, respond with an appropriate error
+	if args.To != nil {
+		if code := stateDb.GetCode(*args.To); len(code) == 0 {
+			return "0x", nil, bind.ErrNonExistentContract
```

Comment:
I had that originally, but I need to return this same error from `bind/backends`, which would create a dependency on `eth` and then I couldn't any more use native daps within `eth` due to circular dependencies.